### PR TITLE
fix: convert numeric options to strings to silence ui-core warnings

### PR DESCRIPTION
**Pull request by edoardo**
And convert the values back to numbers before saving the AO.

### CODE REVIEW

File: packages/app/src/modules/options.js [hunk 1]
```diff
@@ -119,11 +119,13 @@ export const getOptionsFromVisualization = visualization => {
     }
 
     // cast option values from Number for some options
-    ;['sortOrder', 'topLimit'].forEach(option => {
-        if (Object.prototype.hasOwnProperty.call(visualization, option)) {
-            optionsFromVisualization[option] = String(visualization[option])
+    ;['baseLineValue', 'targetLineValue', 'sortOrder', 'topLimit'].forEach(
+        option => {
+            if (Object.prototype.hasOwnProperty.call(visualization, option)) {
```
Comment (amcgee) [inline packages/app/src/modules/options.js]: Why is this not just `if (visualization.hasOwnProperty(option))` ?

File: packages/app/src/modules/current.js [hunk 2]
```diff
@@ -69,11 +69,13 @@ export const getOptionsFromUi = ui => {
     })
 
     // cast option values to Number for some options
-    ;['sortOrder', 'topLimit'].forEach(option => {
-        if (Object.prototype.hasOwnProperty.call(optionsFromUi, option)) {
-            optionsFromUi[option] = Number(optionsFromUi[option])
+    ;['baseLineValue', 'targetLineValue', 'sortOrder', 'topLimit'].forEach(
+        option => {
+            if (Object.prototype.hasOwnProperty.call(optionsFromUi, option)) {
+                optionsFromUi[option] = Number(optionsFromUi[option])
```
Comment (amcgee) [inline packages/app/src/modules/current.js]: Could this ever return `NaN`?

File: packages/app/src/modules/options.js (hunk 1 above)
Comment (edoardo) [reply packages/app/src/modules/options.js]: The linter complains about that form.

File: packages/app/src/modules/current.js (hunk 2 above)
Comment (edoardo) [reply packages/app/src/modules/current.js]: It shouldn't as the UI uses numeric input fields.

File: packages/app/src/modules/options.js (hunk 1 above)
Comment (janhenrikoverland) [reply packages/app/src/modules/options.js]: Do we get e.g. https://eslint.org/docs/rules/no-prototype-builtins from cli-style?

**Pull request merged**